src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -1247,8 +1265,12 @@ const FormCreationForm = ({formUuid, formUrl, formHistoryUrl}) => {
               onChange={onFieldChange}
             />
             <PriceLogic
+              pricingLogic={state.form.pricingLogic}
+              formVariables={state.formVariables}
               rules={state.priceRules}
-              onChange={onPriceRuleChange}
+              // TODO: change both PricingLogic and Rule
+              onChange={onPricingLogicChange}
```

Comment:
You can change the `onChange` callback to:
```javascript
event => dispatch({
      type: 'FIELD_CHANGED',
      payload: {name: 'form.pricingLogic', value: event.target.value},
})
```
Then you can remove the event 'CHANGED_PRICING_LOGIC'